src/Language/ASKEE/Exposure/Interpreter.hs

```diff
@@ -48,14 +50,14 @@ runEval evRead env ev = RWS.runRWST (Except.runExceptT (unEval ev)) evRead env
 
 -------------------------------------------------------------------------------
 
-evalStmts :: [Stmt] -> Env -> IO (Either Text ([DisplayValue], [StmtEff]), Env)
-evalStmts stmts env = evalLoop env stmts
+evalStmts :: [Stmt] -> EvalRead -> Env -> IO (Either Text ([DisplayValue], [StmtEff]), Env)
+evalStmts stmts evr env = evalLoop evr env stmts
```

Comment:
This isn't your fault, but I wonder why we have both `evalStmts` and `evalLoop`, which are essentially the same function. I'd be inclined to remove the former in favor of the latter.

Reply:
Yeah I've wondered the same thing but wasn't super inclined to remove it in this PR. But I don't mind just doing it.

Reply:
I only brought it up since `evalStmts` was the proximal cause of the most recent CI failure for this PR, so now seems like a good time to rip it out :)